### PR TITLE
Add schedule to meetups

### PR DESCRIPTION
**Pull request by bkspace**
Alt Winning

### CODE REVIEW

File: shared/components/EventSchedule/index.js [hunk 1]
```diff
@@ -2,16 +2,14 @@ import React, { PropTypes } from 'react';
 import ScheduleItem from '../ScheduleItem';
 
 const EventSchedule = ({ eventSchedule }) => (
-
-  <article className="EventSchedule">
+  <div className="EventSchedule">
     <h3>Schedule</h3>
-    <dl>
+    <div className="EventSchedule__schedule">
```
Comment (lpil) [inline shared/components/EventSchedule/index.js]: I think `dl` made sense here

https://developer.mozilla.org/en/docs/Web/HTML/Element/dl

**Comment by AndrewBestbier** [comment]
Schedule/Sponsors font-size: 24px;
Schedule details font-size: 18px;
Background color: $event-color-faded (I think)

**Comment by AndrewBestbier** [comment]
Also, you need to add a gif :)

**Comment by lpil** [comment]
Great gif :star:

**Pull request merged**